modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js

```diff
@@ -358,6 +358,11 @@ return L.view.extend({
 				port_opts.push(o);
 			}
 
+			o = s.option(form.Value, 'description', 'Description');
```

Comment:
Wouldn't that better be placed next to the VLAN ID and not after the port list?
Also the `Description` label is not marked as translatable.

Reply:
I'm somewhat torn regarding placement. Having it next to the VLAN ID makes it more visible without scrolling but takes away horizontal space from the actual config area.
Looking at other config interfaces it seems that the description/name is usually placed in the column after the VLAN ID so I'll do that, too.
Translation will be fixed, too